packages/kmod-5.10-nvidia/kmod-5.10-nvidia.spec

```
@@ -0,0 +1,190 @@
+%global kernel_version 5.10.59
+%global nvidia_tesla_470_version 470.57.02
+%global nvidia_tesla_versions %{nvidia_tesla_470_version}
+%global nvidia_kmod_modules nvidia nvidia-modeset nvidia-uvm nvidia-peermem
+%global kernel_sources %{_cross_rootdir}/usr/src/kernels/kernel-devel
+
+Name: %{_cross_os}kmod-5.10-nvidia
+Version: 1.0.0
+Release: 1%{?dist}
+Summary: NVIDIA drivers for the 5.10 kernel
+# TODO: update license to be compliant with the SPDX `LicenseRef-isRef` format
+License: Nvidia
+URL: http://www.nvidia.com/
+
+# NVIDIA .run scripts from 0 to 199
+Source0: https://us.download.nvidia.com/tesla/%{nvidia_version}/NVIDIA-Linux-%{_cross_arch}-%{nvidia_tesla_470_version}.run
+
+# Common NVIDIA conf files from 200 to 299
+Source200: nvidia-tmpfiles.conf.in
+Source202: ldconfig-service.conf
+Source203: modules-load.d.conf
```

nit: to match the convention we use for other drop-ins like sysctls:
```suggestion
Source203: nvidia-dependencies-modules-load.conf
```